samples/drivers/led_tlc59731/boards/stm32wb5mm_dk.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Javad Rahimipetroudi <javad.rahimipetroudi@mind.be>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+
+chip-select {
+	compatible = "gpio-leds";
```

Comment:
Why is this gpio-leds? That makes it looks as if we have two separate LEDs while in reality there is just one.

What we need is a gpio-hog to force enable it. And that should possibly even be in stm32wb5mm_dk.dts, although I don't know if that is possible in a way that it's only enabled if the rgb_leds is enabled.

Alternatively, it should be a named pin in stm32wb5mm_dk.dts (probably better name it GPIO_SELECT2 like in the schematic) and then driven from the example code like is done below.

Reply:
For a GPIO that you use with the `GPIO_DT_SPEC_GET` macro, you don't need a `compatible` at all, because you don't bind it to a driver.